### PR TITLE
Techs

### PR DESCRIPTION
**Pull request by joredudiaz**
## Summary

- Include react element tech

## Jira Story

https://trello.com/c/KMBTFkqx

## Screenshot

image

### CODE REVIEW

File: mock-up/scss/tech.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.techs {
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss]:
```suggestion
.techs-container {
```

File: mock-up/scss/tech.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  align-items: flex-start;
+  display: flex;
+  margin: 0 150px;
+  padding: 30px 80px;
+}
+
+.tech {
+  align-items: center;
+  background-color: #BED23C;
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss]: use color variable

File: mock-up/scss/tech.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  flex-direction: column;
+  height: 325px;
+  padding: 10px;
+  position: relative;
+  width: 200px;
+
+  &::after {
+    content: '';
+    box-sizing: border-box;
+    border: solid #BED23C 1px;
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss]: same

File: mock-up/scss/tech.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 325px;
+  padding: 10px;
+  position: relative;
+  width: 200px;
+
+  &::after {
+    content: '';
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss]: double-quote here

File: mock-up/scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    right: 10px;
+    position: absolute;
+    width: 200px;
+    z-index: -1;
+  }
+}
+
+.tech-img {
+  height: 100px;
+  width: 100px;
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss]: add `object-fit: cover;` propery

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: mock-up/index.html [hunk 6]
```diff
@@ -16,6 +16,15 @@
     <h1 class="title-words">Front-End Tools</h1>
   </div>
 
+  <div class="techs">
+    <div class="tech">
+      <img src="assets/react_icon.svg" alt="react" class="tech-img">
+      <p class="tech-name">ReactJS</p>
+      <p class="tech-release"><strong>Release: 2013</strong></p>
```
Comment (betoobernal) [inline mock-up/index.html]: apply a strong style with css.
`
 font-weight: ...;
`

**Review by FrankIglesias — APPROVED** [review]

**Review by betoobernal — APPROVED** [review]

**Pull request merged**